lib/mvi/responses/id_parser.rb

```diff
@@ -5,35 +5,42 @@ module Responses
     class IdParser
       CORRELATION_ROOT_ID = '2.16.840.1.113883.4.349'
       EDIPI_ROOT_ID = '2.16.840.1.113883.3.42.10001.100001.12'
-      ICN_REGEX = /^\w+\^NI\^\w+\^\w+\^\w+$/.freeze
+      ICN_REGEX = /^\w+\^NI\^200M\^USVHA\^P$/.freeze
       VET360_ASSIGNING_AUTHORITY_ID = '^NI^200M^USVHA'
 
       # MVI correlation id source id relationships:
       # {source id}^{id type}^{assigning facility}^{assigning authority}^{id status}
       # id type - NI = national identifier
       #           PI = patient identifier
-      # id status -  A = active
-      #              P = permanent
-      #              H = deprecated(do not use)
-      #              L = pending local merge (do not use)
-      #              PCE = ??
+
+      # id status -  A = Active (only applies to correlation IDs)
+      #              P = Permanent (only applies to ICNs)
```

Comment:
For clarification:
From my research you can divide the IDs into one of two categories:
* ICN
* Correlation ID

An ICN is the MVI internal ID.
All other IDs are Correlation IDs.  That is, they are IDs that correlate with some other system.

An ICN can have an ID Status of 'P', 'T' or 'D' but we will only see ICNs with a status of 'P' in the IDs returned at the `//controlActProcess/subject/registrationEvent/subject1/patient/id` path in the response XML.  Those are the IDs that are parsed here.
In short, every successful response will have an ICN that matches `/^\w+\^NI\^200M\^USVHA\^P$/`

The other IDs are Correlation IDs.
These IDs can have various ID Types, Assigning Authorities and Assigning Facility.
They can have various status IDs as mentioned below: A, D, M, U, L, PCE, etc.
Only 'A' designates a valid, current ID.